src/goto-programs/goto_function.cpp

```diff
@@ -8,8 +8,13 @@ Date: May 2018
 
 \*******************************************************************/
 
+/// \file
+/// Goto Function
+
 #include "goto_function.h"
 
+/// Get the identifiers of the local variables declared in the function and the
+/// identifiers of the paramters of the function.
```

Comment:
Suggestion: implicitly document the parameters by referring to them in this sentence?